web/router.ex

```diff
@@ -31,6 +42,12 @@ defmodule CoursePlanner.Router do
   end
 
   scope "/", CoursePlanner do
+    pipe_through :protected_api
+
+    resources "/calendar", CalendarController, only: [:index]
```

Comment:
conceptually I think it makes more sense to have
```elixir
resources "/calendar", CalendarController, only: [:show], singleton: true
```
because you don't get a list of calendars